<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>1240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:black;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollBar" name="finger4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>610</y>
      <width>61</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color:black;border-color:white;</string>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="finger0">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>1080</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;border-color:white;</string>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="finger3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>61</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color:black;border-color:white;</string>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="finger2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>630</y>
      <width>61</width>
      <height>441</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;
slider-color:white;

</string>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QScrollBar" name="finger1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>690</y>
      <width>61</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>background-color:black;border-color:white;</string>
    </property>
    <property name="maximum">
     <number>19</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>721</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color:yellow;
</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>721</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>КУЛАК</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>КИСТЬ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string>КОЗА</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
